dialogflow-app.js

```diff
@@ -740,6 +740,54 @@ class DialogflowApp extends AssistantApp {
     return this.doResponse_(response, RESPONSE_CODE_OK);
   }
 
+  /**
+  * Triggers an intent of your choosing by sending a followupEvent from the webhook. 
+  * This method is not sending `speech`, `displayText` JSON objects
+  * as they will be ignored by the system.
+  * A context will not be created, so in order to pass the event data 
+  * you will need to reflect it in the intent created in Dialogflow console.
+  * Each variable in the intent should point to another one defined in the event data
+  * as stated at https://dialogflow.com/docs/events#sending_parameters_in_a_query_request.
+  *
+  * @example
+  * const app = new DialogflowApp({request: request, response: response});
+  * const APPLY_FOR_LICENSE = 'apply-for-license-event';
+  * const DATE_TIME = 'dateTime';
+  * app.askWithFollowupEvent(APPLY_FOR_LICENSE, {
+  *     DATE_TIME: new Date()
+  * });
+  *
+  * @param {string} eventName Name of the event.
+  * @param {Object=} eventData Event JSON object.
+  * @return {null|undefined} Null if the event name is not string or event data 
+  * is not a JSON object.
```

Comment:
Indent JSDoc w/ 4 spaces (e.g. check line 729)

Reply:
@smishra2 I'm fixing that!